crates/turbopack-css/src/module_asset.rs

```diff
@@ -149,7 +153,9 @@ impl ModuleCssAsset {
 
     #[turbo_tasks::function]
     async fn classes(self: Vc<Self>) -> Result<Vc<ModuleCssClasses>> {
-        let inner = self.inner();
+        let Some(inner) = *self.inner().await? else {
+            bail!("inner asset should be CSS processable");
+        };
```

Comment:
All of these could probably be
```suggestion
        let inner = *self.inner().await?.context("inner asset should be CSS processable")?;
```

It also kinda sounds like an error condition, shouldn't it be a result instead of an option?